lib/screens/settings/settings.dart

```diff
@@ -50,6 +50,13 @@ class SettingsScreen extends StatelessWidget {
               layoutModel.isStatsVisible = value;
             },
           ),
+          ListTile(
+            title: const Text('Event Configuration'),
+            // subtitle: const Text("Control which badges are visible"),
```

Comment:
Maybe we should have a simple subtitle to make it clear what event configuration is.

"Control how and which Twitch events are displayed" Names are hard but something like that 😅